controllers/comments.js

```diff
@@ -205,15 +247,15 @@ export default {
       );
 
       if (updatedComment[0] === 0) {
-        return res.status(400).json({
+        return res.status(404).json({
           status: res.statusCode,
-          error: 'Comment to update not found'
+          errorMessage: 'Comment to update not found'
```

Comment:
We decided with the team that we can return error not error message in case an error occurs, it's better if you refactor this.